source/source-connector/fundamentals/specify-schema.txt

```diff
@@ -182,6 +182,23 @@ following options:
    output.format.value=schema
    output.schema.infer.value=true
 
+The source connector can infer schemas for incoming documents that
+contain nested documents stored in arrays. Starting in Version 1.9 of the
+{+connector+}, schema inference does not default to ``string`` type
+assignment if there are differences between nested documents
+described by the following cases:
+
```

Comment:
[suggestion] - This reads a little odd to me because we say that starting in version 1.9 we don't do X, but we don't say what it **does** do starting in 1.9. Perhaps we should add a clarifying statement of what it does do?

Suggestion:
Starting in version 1.9 of the {+connector+}, schema inference will infer the appropriate data type instead of defaulting to the ``string`` type assignment if there are differences between nested documents described by the following cases:

*
Alternatively, this could be broken up into two sentences to make it easier to parse:
If there are differences between nested documents, the {+connector+} defaults to a ``string`` type assignment. Starting in version 1.9 of the {+connector+}, schema inference will infer the correct data type instead of using ``string`` in the following cases: